src/main/webapp/app/atlas/shared/competency-contribution/competency-contribution.component.ts

```diff
@@ -19,8 +21,11 @@ export class CompetencyContributionComponent {
     learningObjectId = input.required<number>();
     isExercise = input.required<boolean>();
 
+    atlasEnabled = false;
+
     private readonly courseCompetencyService = inject(CourseCompetencyService);
     private readonly alertService = inject(AlertService);
+    private readonly profileService = inject(ProfileService);
 
```

Comment:
Minor nitpick, does not need to be changed for me: I think usually the inputs and variables should be defined after constants

We usually have the order constants (icons < injections) < inputs < variables < lifecycles < methods 


```suggestion
    private readonly courseCompetencyService = inject(CourseCompetencyService);
    private readonly alertService = inject(AlertService);
    private readonly profileService = inject(ProfileService);

    atlasEnabled = false;
```